### ISSUE TITLE
OPENTHREAD_CONFIG_PLATFORM_NETIF_ENABLE option for the ot-rfsim platform

### ISSUE THREAD

**Issue by alexis-boisserand-acuity** [EskoDijk/ot-ns#185]
The posix platform has a OPENTHREAD_CONFIG_PLATFORM_NETIF_ENABLE option that, when enabled, creates a TUN driver named wpan0. External applications, can then use this interface to communicate over Thread. This would be useful to easily test some application code over Thread (realtime only I guess). How hard do you think it would be to add this same option to the ot-rfsim platform? This could be used in conjunction with linux network namespaces, so that each node could own its own namespace with a single Thread linux interface.

**Comment by EskoDijk** [comment]
Good idea, issue recreated on: https://github.com/openthread/ot-ns/issues/582 (OTNS2 is now merged there)
